docker/config/mwaa-base-providers-requirements.txt

```diff
@@ -1,8 +1,8 @@
-apache-airflow-providers-amazon==8.16.0
-apache-airflow-providers-celery==3.5.1
-apache-airflow-providers-common-sql==1.10.0
-apache-airflow-providers-ftp==3.7.0
-apache-airflow-providers-http==4.8.0
-apache-airflow-providers-imap==3.5.0
-apache-airflow-providers-postgres==5.10.0
-apache-airflow-providers-sqlite==3.7.0
\ No newline at end of file
+apache-airflow-providers-amazon==8.24.0
+apache-airflow-providers-celery==3.7.2
+apache-airflow-providers-common-sql==1.14.0
+apache-airflow-providers-ftp==3.9.1
+apache-airflow-providers-http==4.11.1
+apache-airflow-providers-imap==3.6.1
+apache-airflow-providers-postgres==5.11.1
+apache-airflow-providers-sqlite==3.8.1
```

Comment:
Shouldn't we add the following as well?
apache-airflow-providers-fab==1.1.1
apache-airflow-providers-smtp==1.7.1

And amazon provider package is with aiobotocore:
apache-airflow-providers-amazon [aiobotocore]==8.24.0

Reply:
Addressed.